include/seqan/align/dp_algorithm_impl.h

```diff
@@ -328,8 +327,8 @@ template <typename TDPScout, typename TDPScoreMatrixNavigator, typename TDPTrace
           typename TSeqVValue, typename TSeqVIterator, typename TScoringScheme, typename TColumnDescriptor, typename TDPProfile>
 inline void
 _computeTrack(TDPScout & scout,
-              TDPScoreMatrixNavigator & dpScoreMatrixNavigator,
-              TDPTraceMatrixNavigator & dpTraceMatrixNavigator,
+              TDPScoreMatrixNavigator & scoreNavi,
```

Comment:
Well, there was no reason to rename it. Probably just some left overs from different tests.
I'll change it to the naming before.